module3/projects/take_home/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Mock Take-Home Challenge
+subtitle: Mod 3 Final Assessment
+layout: page
+---
+
+Your final project is a Take-Home Challenge meant to simulate a task that you might receive during an interview process. The scope is smaller scale and more open ended than other projects you've been assigned. The goal is to complete each part in 8 hours or less for a total of 16 hours or fewer over the course of 2 days. Since there is a time limit and due to the tasks' open ended nature, you should manage your time and plan appropriately. Not only is completing the take-home necessary, but also explaining your process and reasoning behind your decisions are equally important. Use this challenge to practice speaking about your planning and decision making processes around the code you write.
+
+Completion and passing the Take Home Presentation is a promotion requirement. When you present, you should speak as though your assessing instructor is your potential employer. **If you would not be passed on to the next stage of the interview process based on your code and presentation, you will not pass this challenge.**
+
+## Things we'll be looking for:
+
+### Backend API
+- A strong understanding of Rails
+- Ability to create restful routes
+- Demonstration of well-organized code, following OOP
+- Test Driven Development
+- Clear documentation
+
+Try to limit your total time to approximately 8 hours on this portion of the project. Prioritize what is important to completing MVP and demonstrating your capabilities as a developer.
+
+### Frontend
+- A strong understanding of React JS and Front-End best practices
+- Your ability to prioritize for MVP
+- An understanding of basic usability practices and standards
+- A clear hierarchy of information
+- Clean, well thought out code
+
+Try to limit your total time to around 8 hours on this portion of the project. Prioritize what is important to get this working and as close to MVP as possible. Please do not attempt to recreate an entire existing entity's website. We're interested in seeing your ability to write clean code to solve a problem with an understanding of basic UX. We're not looking for fancy UI tricks or hacks.
+
+<section class="note">
+### Note
+
+Your specific prompt will be provided to you during the assessment kickoff.
+</section>
+
+### Prompts
+
+#### Tea Subscription
+- [Part 1: Backend API](./tea_take_home_be)
+- [Part 2: Frontend UI](./tea_take_home_fe)
+
+#### News Reader
+- [Part 1: Backend API](./news_take_home_be)
+- [Part 2: Frontend UI](./news_take_home_fe)
```

Comment:
To my point above - these links could just be removed and then shared on Slack during kick off.

Reply:
Yeah I like that plan